fbw-common/src/systems/shared/src/MsfsDetect.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Hax to determine if we're running under MSFS2024.
+ */
+export function isMsfs2024(): boolean {
+  return window.InputBar.MENU_BUTTON_A === 'KEY_MENU_SR_VALID';
+}
```

Comment:
We'll need a somewhat more robust way of doing this if possible, for now this is fine

Reply:
Doesn’t exist. It won’t be an issue when we have a 2024 specific version.